tasks/move-granules/index.js

```diff
@@ -218,17 +218,20 @@ async function moveGranuleFiles(granulesObject, sourceBucket) {
 *
 * @param {string} cmrFiles - array of objects that include CMR xmls uris and granuleIds
 * @param {Object} granulesObject - an object of the granules where the key is the granuleId
+ * @param {string} distEndpoint - the api distribution endpoint
 * @returns {Promise} promise resolves when all files have been updated
 **/
-async function updateCmrFileAccessURls(cmrFiles, granulesObject) {
+async function updateCmrFileAccessURls(cmrFiles, granulesObject, distEndpoint) {
   for (const cmrFile of cmrFiles) {
     const onlineAccessUrls = cmrFile.metadataObject.Granule.OnlineAccessURLs;
     const granule = granulesObject[cmrFile.granuleId];
 
     const urls = onlineAccessUrls.OnlineAccessURL.map((urlObj) => {
       const filename = path.basename(urlObj.URL);
       const file = granule.files.find((f) => f.name === filename);
-      urlObj.URL = file.filename;
+      const extension = path.join(file.bucket, file.filepath);
+      urlObj.URL = path.join(distEndpoint, extension);
+
```

Comment:
Do we need to do anything else to the `urlObj` to match the example at the top of the bucket brigade recommendations? https://wiki.earthdata.nasa.gov/display/CUMULUS/Bucket+Brigade+Recommendations

Also, do we need to differentiate here between public, protected, and private buckets? I'm thinking of @jennyhliu's comment: https://bugs.earthdata.nasa.gov/browse/CUMULUS-463?focusedCommentId=164862&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-164862